cfme/tests/configure/test_database_ui.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+import pytz
+from dateutil import parser
+
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.log_validator import LogValidator
+
+@pytest.mark.tier(2)
+@pytest.mark.uncollectif(lambda appliance: appliance.version < '5.9')
+# this is work starting 5.9
```

Comment:
**REQUIRED**: I don't like the comment being here, can we have it at the end of the decorator line, one could argue it's not even needed as the decorator tells you that.

Reply:
comment is removed